Miki.Framework.Commands.Permissions/PermissionPipelineStage.cs

```diff
@@ -94,10 +102,10 @@ public class PermissionPipelineStage : IPipelineStage
 			{
 				if(await guildUser.HasPermissionsAsync(GuildPermission.Administrator))
 				{
-					return true;
+					//return true;
```

Comment:
The return statement is commented out, was this temporary for a test? If it's not, couldn't the whole if statement be removed?